### ISSUE TITLE
Transliteration name localization

### ISSUE THREAD

**Issue by musculman** [tilezen/vector-datasource#1891]
*Feature request*

I do not see any option to include transliterated names in the layers.md specification.
There is only one mention to transliterated that I could find, and that is in #1398 

Could we extend the spec to include this option?

I am not sure if simply having something like `name:trans` would help or would make things more complicated. Was this ever considered?

**Comment by nvkelso** [comment]
We've considered this, more as a backfill than anything else. Name is in the local language, but if there's no `name:en` available for many places. Auto detection can be performed for many languages, like Korean, and we've talked about an auto transliteration. 

In Who's On First we'd store that as `kor_latn`. But in Tilezen I think we'd export that directly into `name:en`. 

I don't think there needs to be a new property for it in Tilezen. But tell me more about your use case and if there's alignment here :)

**Comment by musculman** [comment]
So, what we have been doing so far was to store the transliterated version separated from the `official` translation, if you will. So that we can differentiate between the two.
I think the the OSM wiki captures it much better than what I could do, this is a quote with a good example:

> As discussed here and on other pages, many places have names in different languages, like "London", "Londres", "Llundain", "Лондон" for the British Capital. In countries which do not use the latin alphabet, places may or may not have an English version of their name (Moscow for instance has name="Москва" and name:en="Moscow" and int_name="Moscow". What is missing in my view is a transliteration (romanization) of the local name(s), possibly in a dedicated tag. For Moscow we would get translit="Moskva".

from here: https://wiki.openstreetmap.org/wiki/Talk:Map_internationalization
This is the main reason why we used to keep them separated.

**Comment by nvkelso** [comment]
I agree it's best to store the original language name in source script, official translation (into Latin script, sometimes called "romanized"), and (unofficial) transliteration (into Latin script) all in the backend map database. 

Example from OSM:

- https://taginfo.openstreetmap.org/search?q=name%3A
    - 140 468 | name:ko-Latn
- https://www.openstreetmap.org/node/1912365631#map=19/37.56668/126.97829

**Comment by nvkelso** [comment]
Lots of fat fingering, updated comment above with more details.

**Comment by musculman** [comment]
In fact, we do not have a use case in which we would show or need both `name:en` and `name:transliterated` at the same time.
Taking the previous example, we don't have a use case in which we could need at the same time `Moscow` and `Moskva`. We just differentiate them, at least in the database, so that if we ever get the official translation, we can add it to the right field. Perhaps this is enough and there is no need to expose this to the end-user?

**Comment by nvkelso** [comment]
Agree there's not a compelling reason now to show these to end-user / bloat the map tiles by exporting dedicated transliteration name property(ies) at this time. I'm open to documenting that name:* can either be from translation or backfilled from transliteration, though.

**Comment by musculman** [comment]
Yes, I think that makes sense and would keep it easier. Thanks for refocusing the discussion so that it ends up being easier for the customer :)
